Reddit/index.js

```diff
@@ -0,0 +1,128 @@
+const Plugin = module.parent.require('../Structures/Plugin')
+const snekfetch = require("snekfetch")
+
+Object.defineProperty(Array.prototype, "random", {
```

Comment:
I don't like extending arrays because it breaks `for in` calls, which may break other plugins.